books/free-programming-books-subjects.md

```diff
@@ -566,6 +566,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 ### Networking
 
 * [An Introduction to Computer Networks](http://intronetworks.cs.luc.edu) (HTML, PDF, Kindle)
+* [Beej's Guide to Network Programming - Using Internet Sockets](https://www.beej.us/guide/bgnet/) - Brian "Beej Jorgensen" Hall (HTML, PDF)
```

Comment:
we don't duplicate links in multiple files. I think this fits better in Network programming than in "C".  We recently started to add cross-links where this makes sense #5535

Reply:
Ok, that makes sense - I've removed the C-language entry.  I haven't added a cross-link for this but I'll have a read of the proposals in there.